restaurant.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Restaurants Page</title>
+    <link rel= 'stylesheet' href="style.css"/>
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.3.1/dist/leaflet.css" />
+    <script src="https://unpkg.com/leaflet@1.3.1/dist/leaflet.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>
+</head>
+
+<body>
+    <div id = 'title'>Yelpington</div>
+    <div id="map"></div>
+    <div id="container"></div>
+    <button id="back">Go Back</button>
+    <script>
+        let fileId = document.location.hash.slice(1)
+        let container = document.getElementById("container")
+        getJson(fileId)
+
+        function goBack() {
+            document.location.href = "http://localhost:8080/"
+        }
+        function getGeoJson(location) {
```

Comment:
Instead of writing this function multiple times in multiple places you could put it in a separate .js file and either export it, and import it wherever you need i, or add another script with `src="somefile.js"`